Sources/Datadog/Logging/LoggingV2Configuration.swift

```diff
@@ -175,3 +122,122 @@ internal struct LoggingMessageReceiver: FeatureMessageReceiver {
         return true
     }
 }
+
+/// Receiver to consume a Crash Log message as Log.
+internal struct CrashLogReceiver: FeatureMessageReceiver {
+    /// Time provider.
+    let dateProvider: DateProvider
+
+    /// Process messages receives from the bus.
+    ///
+    /// - Parameters:
+    ///   - message: The Feature message
+    ///   - core: The core from which the message is transmitted.
+    func receive(message: FeatureMessage, from core: DatadogCoreProtocol) -> Bool {
+        guard
+            case let .custom(key, attributes) = message, key == LoggingMessageKeys.crash,
+            let report = attributes["report", type: DDCrashReport.self],
+            let context = attributes["context", type: CrashContext.self]
```

Comment:
This still couples Logs (and RUM in other place) with CR. Something we'll need to solve before modules extraction is possible. Nothing to act on right now, just raising the point here.